xbmc/pvr/channels/PVRChannel.h

```diff
@@ -34,7 +35,9 @@ namespace PVR
     friend class CPVRDatabase;
 
   public:
-    explicit CPVRChannel(bool bRadio);
+    static const std::string IMAGE_OWNER_PATTERN;
+
+    CPVRChannel(bool bRadio, const std::string& iconPath = "");
```

Comment:
Default value in a constructor definition. This feels incorrect. It would be cleaner to have 2 constructors, one with iconPath and one without.

Reply:
Okay, will change that.